scripts/install_python38.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+yum install -y openssl-devel bzip2-devel libffi-devel xz-devel wget
+yum groupinstall -y "Development Tools"
+
+wget https://www.python.org/ftp/python/3.8.12/Python-3.8.12.tgz
+tar xvf Python-3.8.12.tgz
+cd Python-3.8*/ || exit
+./configure --enable-optimizations --enable-shared --prefix=/usr LDFLAGS="-Wl,-rpath /usr/lib" 
+make python install
```

Comment:
```suggestion
make python install -j $(nproc)
```
to utilize more cores